sass/components/_popover.scss

```diff
@@ -166,9 +166,12 @@
     .popover__row {
         margin: 0 -15px 0;
         padding: 9px 15px 0;
+        .app__body & {
+            border-color: rgba(var(--center-channel-color-rgb), 0.2);
+        }
```

Comment:
We don't need this one anymore, the addition of the color to the .first selector is sufficient ... my bad so I'm removing! 😛
```suggestion
```